frontend/common/src/components/Link/IconLink.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Style } from "util";
```

Comment:
this is unused, intentional?

Reply:
Not sure how that got there 😕 .

Resolved with [remove unused Style import](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2710/commits/2b2ab3c0dd3744d49ebc3e6b6b2bbb2e8967ceff)